tools/topology/topology2/include/components/asrc.conf

```diff
@@ -29,6 +29,12 @@ Class.Widget."asrc" {
 	#include common component definition
 	<include/components/widget-common.conf>
 
+	#include platform specific module configs for the asrc module
```

Comment:
@jsarha is there value in this commit just yet? Can we not add the files as and when needed?

Reply:
@lgirdwood Can you ack as this was an ask from you? I.e. to have the placeholders or not.

Reply:
1. I agree add on demand.
2. if want to add in this patch, I would suggest there is a folder to cover all component instead of directly under platform/intel/mod_cfg/